src/sysl/importers/import_swagger.py

```diff
@@ -273,12 +281,9 @@ def info_attrs(info, prefix=''):
             w('!type {}:', tname)
 
             with w.indent():
-
-                tspec_items = tspec.get('properties')
-
-                if tspec_items:
-                    for (fname, fspec) in sorted(tspec_items.iteritems()):
-
+                properties = extract_properties(tspec)
+                if properties:
```

Comment:
This `if` is no longer needed as you have an empty dict instead of `None`. The iteration will be a no-op now.

Reply:
i agree, i removed that initially, then noticed hte code is doing `if properties: ... else: .... `so reverted it